contracts/token/ERC721/ERC721.sol

```diff
@@ -427,12 +423,15 @@ abstract contract ERC721 is Context, ERC165, IERC721, IERC721Metadata, IERC721Er
     }
 
     /**
-     * @dev Reverts if the `tokenId` has not been minted yet.
+     * @dev Reverts if the `tokenId` doesn't have a current owner (it hasn't been minted, or it has been burned).
+     * Returns the owner.
      */
-    function _requireMinted(uint256 tokenId) internal view virtual {
-        if (_ownerOf(tokenId) == address(0)) {
+    function _requireOwned(uint256 tokenId) internal view virtual returns (address) {
```

Comment:
Should this be virtual ?

Reply:
I don't see why not.

Reply:
I'm wondering what would happen if someone overrides this to change the returned value instead of overriding `_ownerOf`.

Reply:
Oh I see what you mean. Alright, I'm removing virtual.